example/App.js

```diff
@@ -202,6 +202,7 @@ const App = ({}) => {
           startHour={15}
           fixedHorizontally={showFixedComponent}
           showTitle={!showFixedComponent}
+          timesColumnWidth={0.2}
```

Comment:
Should the name be clearer?

Reply:
Sure it can be better. Is the best I came up with 😅

Do u have something in mind? What u think is not clear from the name?

Reply:
I meant the name of the prop `timesColumnWidth`, it makes me think I should put the fixed width value, but actually, it's the ratio

Reply:
I guess `timesColumnWidthRatio` could be the name, WDYT, @pdpino ?

Reply:
It can be provided either as a ratio (`0 < value < 1`) or as fixed width value (`1 <= value`)

Reply:
I see, I missed this line, https://github.com/hoangnm/react-native-week-view/pull/253/files#diff-accc88f2dd24cd8f77ae72d8b3871d3eb26c0b086886fb5e8ba03fc65200dcfaR108, but what if the value is over 100?

Reply:
(sorry the late reply)

The code is wrong, is not doing what I intended. Good catch.

I'll fix this asap and let you know when it's ready

Reply:
It looks great!